forum/investors/models.py

```diff
@@ -52,4 +56,34 @@ class Meta:
         verbose_name_plural = 'Investors'
 
     def __str__(self):
-        return self.company_name
\ No newline at end of file
+        return self.company_name
+
+
+class InvestorStartup(models.Model):
+    """
+    Represents a many-to-many relationship between investors and startups, 
+    indicating which investors are following which startups.
+
+    Attributes:
+        investor_id (ForeignKey): A reference to the User model, representing 
+        the investor who follows a startup. The relationship is set to cascade 
+        on deletion, meaning if the user is deleted, their follows will also be removed.
+
+        startup_id (ForeignKey): A reference to the Startup model, representing 
+        the startup being followed by the investor. Like the investor, this relationship 
+        also cascades on deletion.
+
+        saved_at (DateTimeField): A timestamp indicating when the investor 
+        started following the startup. This field is automatically set to 
+        the current date and time when the relationship is created.
+    """
+
+    investor_id = models.ForeignKey(User, on_delete=models.CASCADE, related_name='investor_startups')
+    startup_id = models.ForeignKey(Startup, on_delete=models.CASCADE, related_name='startup_investors')
+    saved_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = ('investor_id', 'startup_id')
```

Comment:
You are using `unique_together` to enforce uniqueness for the combination of `investor_id` and `startup_id`. Since Django 2.2, it’s recommended to use the `UniqueConstraint` instead of `unique_together` for clarity and future-proofing.